arm/.global/global.module.tests.ps1

```diff
@@ -763,6 +763,19 @@ Describe 'Deployment template tests' -Tag Template {
                 $TenantIdKeyCount -eq $TenantIdValueCount | Should -Be $true -Because ("Parameter file should not contain the Tenant ID guid, instead should reference a token value '$ParameterFileTokenName'")
             }
         }
+
+        It '[<moduleFolderName>] [Tokens] Parameter files, resource name should contain the namePrefix token' -TestCases $deploymentFolderTestCases {
```

Comment:
```suggestion
        It '[<moduleFolderName>] [Tokens] Parameter files should use the namePrefix token where possible' -TestCases $deploymentFolderTestCases {
```

Reply:
Will be removed from the tests and created as a backlog item